common/test/acceptance/pages/studio/home.py

```diff
@@ -0,0 +1,18 @@
+"""
+Home page for Studio when logged in.
+"""
+from bok_choy.page_object import PageObject
+
+from common.test.acceptance.pages.studio import BASE_URL
+
+
+class HomePage(PageObject):
+    """
+    Home page for Studio when logged in.
+    """
+
+    url = BASE_URL + "/home"
+
+    def is_browser_on_page(self):
+        return all([self.q(css='.page-header').visible,
+                    'Studio Home' == self.q(css='.page-header')[0].text])
```

Comment:
I don't think this will work. The q() returns a BrowserQuery object not an iterable, so you wouldn't use [0] to specify the 1st element of it.
I think what you mean is self.q(css='.page-header').text[0]